### PR TITLE
[feature] Support clean CRI 

### LINKED ISSUE #1415: Add CRI clean logic when delete all the cluster
### Your current KubeKey version

v2.1.0

### Describe this feature

Sometimes, I want to clean all CRI(docker/containerd...) configuration and data directories when I delete the total cluster.

### Describe the solution you'd like

./kk delete cluster -f config.yaml --all

Add --all params to indicate remove all CRI  directories.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by life-**
### What type of PR is this?
/kind feature

### What this PR does / why we need it:
Sometimes, I need to delete all cri(docker/containerd) related configuration and data direcoties when delete the total cluster.

### Which issue(s) this PR fixes:

Fixes #1415 

### Special notes for reviewers:
```
```

### Does this PR introduced a user-facing change?

```release-note

```

### Additional documentation, usage docs, etc.:

```docs
./kk delete cluster -f config.yaml --all
./kk delete cluster -f config.yaml -a
```

### CODE REVIEW

**Comment by life-** [comment]
some files changed (especially comments) after gofmt  executed automatically.

File: cmd/ctl/delete/delete_cluster.go [hunk 1]
```diff
@@ -57,11 +58,13 @@ func (o *DeleteClusterOptions) Run() error {
...
 		KubernetesVersion: o.Kubernetes,
+		DeleteCRI:         o.DeleteCRI,
 	}
 	return pipelines.DeleteCluster(arg)
 }
 
 func (o *DeleteClusterOptions) AddFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVarP(&o.ClusterCfgFile, "filename", "f", "", "Path to a configuration file")
 	cmd.Flags().StringVarP(&o.Kubernetes, "with-kubernetes", "", "", "Specify a supported version of kubernetes")
+	cmd.Flags().BoolVarP(&o.DeleteCRI, "all", "a", false, "Delete total cri conficutation and data directories")
```
Comment (24sama) [inline cmd/ctl/delete/delete_cluster.go:69]: Prefer to use `-A`

**Comment by 24sama** [comment]
Hi @life- @zaunist , thanks!

**Comment by life-** [comment]
@24sama Thanks for your code review.  I have already optimized the parameter from -a to -A.

**Comment by 24sama** [comment]
Thanks again!

/lgtm

**Comment by 24sama** [comment]
@life- It would be better if you can do the following things:
1. Please create another PR to complement this [document](https://github.com/kubesphere/kubekey/blob/master/docs/commands/kk-delete-cluster.md).
2. Please squash your commits

**Comment by life-** [comment]
@24sama I have done the work as your suggestion! Please take your time to review the changes.

BTW, Thanks for your good suggestion!

**Pull request merged**